src/script/src/python/coprocessor.rs

```diff
@@ -450,6 +451,8 @@ pub(crate) fn exec_with_cached_vm(
         // set arguments with given name and values
         let scope = vm.new_scope_with_builtins();
         set_items_in_scope(&scope, vm, &copr.deco_args.arg_names, args)?;
+        crate::python::dataframe::data_frame::set_dataframe_in_scope(&scope, vm, "df", rb)?;
```

Comment:
Is the name "df" too commonly used to conflict with others?

Reply:
> Is the name "df" too commonly used to conflict with others?

How about just "dataframe"?

Reply:
"dataframe" is good to me